src/form-components/RangeEditor.js

```diff
@@ -11,8 +11,8 @@ export default class RangeEditor extends React.Component {
     render() {
         return <React.Fragment>
             <div className="form-row slider-wrapper">
-                <label className="mb-0 w-100" htmlFor={this.props.id}>
-                    <em>{this.props.itemlabel}</em>
+                <label key="dataId" className="mb-0 w-100" htmlFor={this.props.id}>
+                    {this.props.itemlabel.map((value, index) => <em key={index}>{value}</em>)}
```

Comment:
Hmmm, note that if `value` is the `<strong>α</strong>` tag, this anonymous function will then return `<em><strong>α</strong></em>`, which is kind of awkward.

Reply:
Isn't it just like using bold and italics together in a text editor? If you feel that the <em> tag is unnecessary I'll remove it.

Reply:
`<em>` tag apparently works in this text editor @_@''

Reply:
Ahh - okay, I _**finally**_ understand what you mean - Yeah. The resulting string is both italicized and bold, which is what we want anyways! Makes sense.